addons/l10n_ar_withholding/demo/account_tax_demo.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <function model="account.tax" name="write">
+        <function eval="[[('l10n_ar_withholding', '!=', False)]]" model="account.tax" name="search"/>
+        <value eval="{'amount_type': 'percent', 'amount' : 1}"/>
```

Comment:
The idea to have it only on demo, is that for production use cases,the user needs to check and put the right percentage.
So that's why we've decided to keep 0 on data but put a dummy percentage for demo purpose

cc @vbe-odoo